beacon_node/network/src/sync/block_lookups/single_block_lookup.rs

```diff
@@ -302,8 +312,16 @@ impl<const MAX_ATTEMPTS: u8, T: BeaconChainTypes> SingleBlockLookup<MAX_ATTEMPTS
                         downloaded_block,
                         downloaded_blobs,
                     } = components;
-                    downloaded_block.as_ref().map(|block| {
-                        BlockWrapper::BlockAndBlobs(block.clone(), std::mem::take(downloaded_blobs))
+                    downloaded_block.as_ref().and_then(|block| {
+                        //TODO(sean) figure out how to properly deal with a consistency error here,
```

Comment:
iiuc, the peer hasn't sent us all the requested components here so makes sense to downscore

Reply:
I'll try to address this in the networking refactor PR, or maybe a followup